wpilib/wpilib/serialport.py

```diff
@@ -205,6 +205,7 @@ def write(self, buffer: bytes) -> int:
         """Write raw bytes to the serial port.
         
         :param buffer: The buffer of bytes to write.
+
         Python-Specific: No count parameter needed
```

Comment:
This really shouldn't be in the docstring, methinks.

Reply:
Yeah, it's an implementation note for future porters, users shouldn't care.

Reply:
ok. Below the docstring in the method, or class-level above the method?

Reply:
In the method below the docstring.